docs/security/trust/content_trust.md

```diff
@@ -267,13 +268,18 @@ Because the tag `docker/cliffs:latest` is not trusted, the `pull` fails.
 A user who wants to disable content trust for a particular operation can use the
```

Comment:
Yes, if it makes sense - since both parts seem to cover the same flag, and if the section is combined it would mean that `latest` can get pushed without trust.